SwiftStyleGuide.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SwiftStyleGuide.swift
+//  PurchasesCoreSwift
+//
+//  Created by Andrés Boedo on 7/12/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
```

Comment:
Add `imports should be alphabetized`
- I think swiftlint might be able to take care of that.

Reply:
+100

Reply:
[swiftlint rule](https://realm.github.io/SwiftLint/sorted_imports.html) added